documentation/modules/rn-2.2.adoc

```diff
@@ -39,3 +51,10 @@ Retaining the `Importer` pod for debug purposes causes warm migration to hang du
 As a temporary workaround, the `Importer` pod is removed at the end of the precopy stage so that the precopy succeeds. However, this means that the `Importer` pod log is not retained after warm migration is complete. You can only view the `Importer` pod log by using the `oc logs -f <cdi-importer_pod>` command during the precopy stage.
 
 This issue only affects the `Importer` pod log and warm migration. Cold migration and the `virt-v2v` logs are not affected.
+
+.Migration plans for VMs with no operation system fail with an unclear error message.
+
+When migration plans for VMs with no operation system fail, the error message does not indicate the reason for the failure. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2008846[*BZ#2008846*])
+
+.Migration plans created from the CLI using names instead of IDs for network and storage mapping and for VMs become corrupted when opened in the {product-title} web console.
+If you create a migration plan from the CLI and use names for storage and network mapping and for VMs instead of their IDs, the plan becomes corrupted when you open it in the {product-title} web console. The mapping is invalid, and the edit and duplicate features do not work.(link:https://bugzilla.redhat.com/show_bug.cgi?id=2023801[*BZ#2023801*])
```

Comment:
Add space between the period and the parenthesis.
Fix attribute for MTV.

Remember to use the right BZ. You need to point to the engineering bug, not your docs bug.

"Corrupted" is not the right word. That implies that the plan shows gibberish. It's not the plan that shows the problems. It's the Migration plan wizard. The wizard shows the network and storage as "unavailable". It can't find them. Check the screenshots in the engineering bug.